spec/models/course_spec.rb

```diff
@@ -1865,16 +1865,43 @@
   end
 
   context "bursaries and scholarships" do
-    let!(:financial_incentive) { create(:financial_incentive, subject: modern_languages, bursary_amount: 255, scholarship: 1415, early_career_payments: 32) }
+    let(:english) { find_or_create(:secondary_subject, :english) }
+    let(:maths) { find_or_create(:secondary_subject, :mathematics) }
+    let(:religious_education) { find_or_create(:secondary_subject, :religious_education) }
+    let!(:financial_incentive) { create(:financial_incentive, subject: english, bursary_amount: 255, scholarship: 1415, early_career_payments: 32) }
 
-    subject { create(:course, :skip_validate, level: "secondary", subjects: [modern_languages]) }
```

Comment:
Did it break modern language?

Can we have a test that reflects this kinda course
https://www.find-postgraduate-teacher-training.service.gov.uk/course/2KL/J532